lib/mumukit/auth/role.rb

```diff
@@ -64,6 +95,10 @@ class Owner < Role
       def parent_allows?(*)
         false
       end
+
+      def narrower_than_other?(*)
```

Comment:
to keep consistency this should be private

Reply:
You are right!